Dev1.java

```diff
@@ -0,0 +1,12 @@
+public class Dev1 {
+    public static void main(String[] args) {
+	for (int n = 0; n <= 100; n++) {
+	    if (n % 3 == 0) {
+	        System.out.println("3*" + n/3);
+	    } else {
+	        System.out.printl(n);

```

Comment:
Have you tried to compile this? I doubt if System.out has printl (not println) so I suppose there is a compilation issue